src/components/ArticlesTable/ArticlesTable.js

```diff
@@ -91,11 +91,11 @@ export class ArticlesTable extends Component {
                     >
                       <MenuItem
                         disabled
-                        primaryText={`Saved: ${moment(item.get('timestamp')).format('lll')}`}
+                        primaryText={`Saved: ${moment.utc(item.get('timestamp')).local().format('lll')}`}
```

Comment:
I think it would be good to factor out common time manipulation routines, so we have a single place to change in the future. I know we only have two instances now, but we will have more in the future.